applications/forms/hacker.py

```diff
@@ -79,6 +79,11 @@ def clean_projects(self):
             )
         return data
 
+    def clean_site(self):
+        data = self.cleaned_data["site"]
+        validate_url(data, "website")
```

Comment:
diria que amb la string "website" buscarà si la web introduïda conté la paraula "website", pot no passar sempre

Reply:
No. En cas de no ser una web vàlida tira l'error de Please enter a valid webside...
Si vols prova correr en local i posa un camp random a site

Reply:
okay just to know, desconeixia aquesta feature jeje ho vaig mirar des del mobil xd